### PR TITLE
Add minimal Skynode RC13 config to the PAB manifest

### PR DESCRIPTION
**Pull request by niklaut**
Adds a new baseboard revision for a Skynode RC13 baseboard without PX4IO, CAN2, CAN3, PM2.

### Changelog Entry

For release notes:
```
Feature: Add Skynode RC13 baseboard with many parts removed
```

### Test coverage

Yes. SBUS input has been tested.

### CODE REVIEW

**Review by davids5 — CHANGES_REQUESTED** [review]
@niklaut There is no code point 11 decimal. If the extend ID from EEPROM is used 0x010 is the minimum value. So I think you need  0x011 (17 decimal)

**Comment by niklaut** [comment]
I thought I was using 0x011 (17 decimal) in this code?

File: boards/px4/fmu-v5x/init/rc.board_defaults [hunk 1]
```diff
@@ -16,7 +16,7 @@ param set-default SENS_EN_INA238 0
 param set-default SENS_EN_INA228 0
 param set-default SENS_EN_INA226 1
 
-if ver hwbasecmp 008 009 00a 010
+if ver hwbasecmp 008 009 00a 010 011
```
Comment (davids5) [inline boards/px4/fmu-v5x/init/rc.board_defaults:19]: Sorry @niklaut I should have been clearer : I was revering to here.

**Review by davids5 — CHANGES_REQUESTED** [review]

File: boards/px4/fmu-v5x/init/rc.board_defaults (hunk 1 above)
Comment (niklaut) [reply boards/px4/fmu-v5x/init/rc.board_defaults:19]: ah, so `hwbasecmp` expects decimal values? Cos then `00a` is wrong too. That's why assumed it was hexadecimal.

Comment (davids5) [reply boards/px4/fmu-v5x/init/rc.board_defaults:19]: better check it.

Comment (davids5) [reply boards/px4/fmu-v5x/init/rc.board_defaults:19]: it my be hex

Comment (niklaut) [reply boards/px4/fmu-v5x/init/rc.board_defaults:19]: Are you sure it's not hexadecimal? The `ver hwbasecmp` does a `strcmp`, so it takes a string as input:
https://github.com/PX4/PX4-Autopilot/blob/28db3e1c8ca97fd1bb6067be15c9caf19db8e735/src/systemcmds/ver/ver.cpp#L147

It compares against `px4_board_base_type()` which resolves to the `hw_base_info` char array.
https://github.com/PX4/PX4-Autopilot/blob/28db3e1c8ca97fd1bb6067be15c9caf19db8e735/platforms/nuttx/src/px4/nxp/imxrt/board_hw_info/board_hw_rev_ver.c#L302

which is formatted here
https://github.com/PX4/PX4-Autopilot/blob/28db3e1c8ca97fd1bb6067be15c9caf19db8e735/platforms/nuttx/src/px4/nxp/imxrt/board_hw_info/board_hw_rev_ver.c#L409

with a `%03x` specifier:
https://github.com/PX4/PX4-Autopilot/blob/28db3e1c8ca97fd1bb6067be15c9caf19db8e735/platforms/nuttx/src/px4/common/include/px4_platform/board_determine_hw_info.h#L41

https://github.com/PX4/PX4-Autopilot/blob/28db3e1c8ca97fd1bb6067be15c9caf19db8e735/platforms/common/include/px4_platform_common/board_common.h#L285

So I think this is correct, the code does the right thing and the debugger also agrees with me:
```
(gdb) p hw_base_info
$1 = "011\000\000\000\000\000\000"
```

Comment (davids5) [reply boards/px4/fmu-v5x/init/rc.board_defaults:19]: OK - I wrote it and did not remenber!

**Review by davids5 — APPROVED** [review]

File: boards/px4/fmu-v5x/init/rc.board_defaults (hunk 1 above)
Comment (niklaut) [reply boards/px4/fmu-v5x/init/rc.board_defaults:19]: happens to the best of us ;-P

**Pull request merged**